Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -520,7 +520,15 @@ public final class PackageBuilder {
                 if subpath == "" || subpath == "." {
                     return packagePath
                 }
-                let path = packagePath.appending(RelativePath(subpath))
+
+                let path: AbsolutePath
+
+                if subpath.hasPrefix("/") {
+                    path = AbsolutePath(subpath)
+                } else {
+                    path = AbsolutePath(subpath, relativeTo: packagePath)
+                }
```

Comment:
Because this is a user input, I was thinking, we should get a method which takes a string and tells if it is a valid absolute path, a valid relative path, or an invalid path.